tests/unittests/net/network_configs.py

```diff
@@ -420,6 +421,7 @@
                 [ipv4]
                 method=auto
                 may-fail=false
+                route-metric=10000
                 route1=0.0.0.0/0,65.61.151.37
```

Comment:
If the metric value were added here, it would have been the same effect since it also specifies default route. I think we should do it this way instead of adding `route-metric` . In other words, there is no need to check for default route and add `route-metric`. All matric values can be added as a comma separated value.

Reply:
Adding route-metric for the default route, both sysconfig and NM are identical.

Reply:
OK fine then.